great_expectations/data_context/types/base.py

```diff
@@ -191,7 +190,7 @@ class Meta:
         unknown = INCLUDE
 
     class_name = fields.String(required=True)
-    module_name = fields.String(allow_none=True)
+    module_name = fields.String(missing="great_expectations.datasource")
```

Comment:
I have a preference for adjusting to completely omit the key from the config when it has the default value, and also to ensure we don't have defaults specified in multiple places (there is an existing default set via `instantiate_class_from_config`)

Reply:
Couldn't figure out quick way to exclude empty keys from yaml so keeping filled in defaults for now, but nixed that default specification in instantiate_class_from_config.